### DISCUSSION TITLE
Z0emmc

### DISCUSSION THREAD

**Discussion by marcelk1101** [devbis/z03mmc#89] [Q&A]
I'm using Firmware z03mmc for LYWSD03MMC devices. Thanks for developing!
Is it possible to get the temperatur on mqtt with decimal, for example 18,7°C ? 

![image](https://github.com/devbis/z03mmc/assets/83179131/04a87184-1e54-481c-942a-1e07972eb362)

**EDIT:** 2.Question: After flashing with z03mmc I need to know how to connect to the device?

**Comment by devbis** [comment]
Temperature and Humidity are sent with 2 decimal places, like 23.45º 
Any other conversion is done by your software. (Tuya gateways is known to round values)

To connect it to your coordinator, enable pairing mode and short reset and gnd points near the battery socket for 3 seconds

**Comment by marcelk1101** [comment]
Thank you. In my Smartphone app (smart life) and my Server (iobroker) there aren't decimal places.

Do I need to pair it again and need to manually configure it?

I cant see battery values too...

![Screenshot_2023-12-11-09-49-31-621_com tuya smartlife](https://github.com/devbis/z03mmc/assets/83179131/4e3e1436-1dfc-486f-8464-20b3de03d55f)

**Comment by devbis** [comment]
As far as I know, tuya dosn't allow configuring it and this firmware is not registered in tuya as it is not related to it in any way. 
You may switch to a different zigbee coordinator harware+software (e.g. zigbee2mqtt)  or use a different thermometer to use with tuya correctly. 

Issue is here https://github.com/devbis/z03mmc/issues/64
